### ISSUE TITLE
Risk level for all attendees at an event

### ISSUE THREAD

**Issue by neurodynamic** [microcovid/microcovid#957]
I'd love to see a way to do a broader type of calculation for events. Something like: if I hold an outdoor event with 200 people, what is the risk that any one person (rather than the risk to just me personally) might get covid. What about an indoor event with 10 people, or an outdoor, distanced event with 1000 people, etc.

**Comment by blanchardjeremy** [comment]
Can you say more about the purpose/value of doing that calculation? How does knowing the likelihood that one person has covid help?

**Comment by blanchardjeremy** [comment]
This isn't quite what you're asking, but I thought I'd link it here anyway. This is the "what happens if one person who does have COVID walks into a room?" calculator: https://indoor-covid-safety.herokuapp.com/

**Comment by jy2wong** [comment]
This functionality exists in the Jimenez spreadsheet at https://docs.google.com/spreadsheets/d/16K1OQkLD4BjgBdO8ePj6ytf-RpPMlJ6aXFg3PrIQBbQ/edit#gid=996535083 (look for "Conditional Results for ALL ATTENDEES & ONE EVENT (assuming number of infected above, typically 1)") but it's not quite as friendly as the microcovid calculator.

**Comment by jy2wong** [comment]
You can kiiiinda use the microcovid "adjusted prevalence", multiply that by number of projected attendees, and multiply *that* against the Jimenez conditional results number to get "how many people might get infected".

**Comment by jy2wong** [comment]
@neurodynamic I found another calculator that covers what the Jimenez spreadsheet provides (in an easier to use format). You'll still need to combine the outputs from this calculator and the adjusted prevalence from the microcovid calculator, though.

**Comment by neurodynamic** [comment]
@jy2wong Ooh interesting; thanks!

**Comment by beshaya** [comment]
> Not the likelihood that one person has covid, but the likelihood that a in a group of X people under X conditions, how many (if any) are likely to be infected when we don't know if anyone has covid or not.

You can get this out of microCOVID with a little extra work:
![image](https://user-images.githubusercontent.com/5877421/129782246-c07d3004-6bee-49a1-b1ba-2f6d1c128f43.png)

The expected number of people who will get COVID as a result of the event is the microCOVID cost of the event times the number of people.
In the example, thats 8000microCOVID * 100 people = 800,000microCOVID = 0.8 expected infections as a result.

The Jimenez conditional simply does this multiplication (assuming 1 person coming to the event is infected)

The Max-plank tool calculates "Chance that at least one person gets infected" as (1 - chance_one_person_gets_infected) ^ number_of_participants. That formulation is incorrectly assumes that the chance of each person getting infected is independent from each other - COVID-19 is an over-dispersed pathogen, which means that you tend to either get no infections or lots of infections (i.e., in the default example there's a 10% chance that any specific individual gets COVID, but the probability distribution for the group is more like, if anyone has COVID, there's a 5% chance that everyone gets COVID , a 50% chance that no one gets COVID, and the remaining probability distributed in between.) The Atlantic has a good [article](https://www.theatlantic.com/health/archive/2020/09/k-overlooked-variable-driving-pandemic/616548/) on this

In other words, if you run the example event 1000 times, you'd expect 800 infections, but it would probably come from ~20 people infected at each of ~40 events rather than 1 infection from each of 800 of the events.
